### DISCUSSION TITLE
esp32-2432s022

### DISCUSSION THREAD

**Discussion by mstewart197930** [BitMaker-hub/NerdMiner_v2#315] [Q&A]
HI, I have the above mentioned board, i mistakenly ordered the wrong screen size when ordering, but this one just gives a blank white screen on boot, the firmware is running as i can see the AP,
is it easy enough for me to modify for the screen difference?
i believe it is a different display chip to the 28r....

**Comment by gregcasey14910** [comment]
Similar issue -- After getting a 2432S028 - I got it to work, with the libraries for the Cheap Yellow Display Board.   It appears to be a eSPI dirven display.   FROM WHAT I have gathered, the 2432s022 uses a ST7789 display but in the raw parrallel  Display device.  with a x8 bus going back to the ESP32.   Has anyone seen a display library that can drive this parrallel display bus.   OR am I totally incorrect ??  Greg

**Reply by gregcasey14910** [reply]
My comments about the parrallel are for the S022 NOT S028 ( clearly an eSPI )

**Reply by paelzer** [reply]
"TFT_eSPI" library by Frank Bodmer, which is actually used for many displays on the NerdMiner boards can deal with displays with parallel bus as well.

https://github.com/Bodmer/TFT_eSPI

Find documentation and configuration options in "User_Setup.h" file in the library.

For example, this 3.5" parallel one works fine with TFT_eSPI:
![image](https://github.com/BitMaker-hub/NerdMiner_v2/assets/16374477/f077ee07-9e56-499a-95bf-571bda638926)

**Reply by mesarim** [reply]
I was looking at the same, and found that someone wrote a script which works using pretty much this library: bb_spi_lcd.h - that works pretty much out of the box.  If you found anything else I'm also interested in what this lil cyd can do.
